lib/tasks/rake_helpers/archived_claims.rb

```diff
@@ -0,0 +1,48 @@
+module RakeHelpers
+  module ArchivedClaims
+    def self.write filename
+      CSV.open(filename, 'wb') do |csv|
+        csv << [
+          'Claim type',
+          'Case number',
+          'Advocate/Litigator',
+          'Main dependant',
+          'MAAT',
+          'Total (inc VAT)',
+          'Status',
+          'Case type',
+          'Submitted',
+          'Archived date'
+        ]
+
+        rows = query
```

Comment:
not sure how many claims this would return in a production like environment but if this is a lot we should probably use a `find_each` to retrieve and write in batches - see [find_each](https://api.rubyonrails.org/classes/ActiveRecord/Batches.html#method-i-find_each) for details.